runtime/imports.go

```diff
@@ -354,8 +356,7 @@ func ext_blake2_256_enumerated_trie_root(context unsafe.Pointer, valuesData, len
 	instanceContext := wasm.IntoInstanceContext(context)
 	memory := instanceContext.Memory().Data()
 
-	runtimeCtx := instanceContext.Data().(*RuntimeCtx)
-	s := runtimeCtx.storage
+	t := &trie.Trie{}
```

Comment:
I am concerned with this usage. We havent really made it possible to use the trie like this, it's missing DB and children trie map. Ideally we would have a simple version of the trie that is purely in memory. There might be little/no harm in merging this, but we should definitely address this in the near future.

This will be exacerbated when we start trying to cache entries of the trie.

Reply:
I think this is fine, we could do `trie.NewEmptyTrie` alternatively to initialize the child stuff, but I don't think it would ever be needed in this case. the enumerated trie is supposed to be temporarily constructed.